lib/links.js

```diff
@@ -4,6 +4,7 @@ const FIXES_RE = /(Close[ds]?|Fix(e[ds])?|Resolve[sd]?)\s*:\s*(\S+)/mgi;
 const FIX_RE = /(Close[ds]?|Fix(e[ds])?|Resolve[sd]?)\s*:\s*(\S+)/i;
 const REFS_RE = /Refs?\s*:\s*(\S+)/mgi;
 const REF_RE = /Refs?\s*:\s*(\S+)/i;
+const PR_RE = /PR-URL?\s*:\s*(\S+)/i;
```

Comment:
the `?` is a copy-paste error, in the lines above `Ref` or `Refs` are both valid, but `PR-UR:`  without the `L` is not.